src/components/my-widgets-page.jsx

```diff
@@ -215,7 +215,7 @@ const MyWidgetsPage = () => {
 				widgetName: inst.widget.name,
 				dir: inst.widget.dir,
 				successFunc: (data) => {
-					if (!data || (data.type == 'error'))
+					if (data && (data.type == 'error'))
```

Comment:
The lack of a null check here is probably fine since this is coming from the `useDeleteWidget` mutation which handles errors.